src/views/AddItem.jsx

```diff
@@ -19,6 +19,12 @@ export function AddItem({ listToken }) {
 	// Notify when adding item is unsuccessful
 	const notifyFailedRequest = () => toast.error('Failed to add item');
 
+	// Notify if the item already exists in db
+	const notifyInList = () => toast.error('Already in your list');
```

Comment:
In the future, you may want to consider making this message a full sentence. A future improvement might also be passing the value the user entered into `notifyInList` so that it says "It looks like {thing} is already in your list".